### ISSUE TITLE
Adding dct:accessRights to DCAT

### ISSUE THREAD

**Issue by andrea-perego** [w3c/dxwg#160]
> Related use case: [5.17 Data access restrictions](https://www.w3.org/TR/dcat-ucr/#ID17)
> Related requirement: [6.7.1 Dataset access [RDSA]](https://www.w3.org/TR/dcat-ucr/#RDSA)
> Related action: [ACTION-86](https://www.w3.org/2017/dxwg/track/actions/86)

The relevant use case provides the context, and the existing solutions. 

I copy-paste the relevant problem statement for our convenience:

> The types of possible access restrictions of a dataset are one of the key filtering criteria for data consumers. For instance, while searching in a data catalogue, I may not be interested in those data I cannot access (closed data), or in those data requiring I provide personal information (as data that can be accessible by anyone, but only after registration).
>
> Moreover, it is often the case that different distributions of the same dataset are released with different access restrictions. For instance, a dataset containing sensitive information (as personal data) should not be publicly accessible, although it would be possible to openly release a distribution where these data are aggregated and/or anonymized.
>
> Finally, whenever data are not publicly available, an explanation of a reason why they are closed should be provided - especially when these data are maintained by public authorities, or are the outcomes of public-funded research activities.

[The existing solutions described in the UC](https://www.w3.org/TR/dcat-ucr/#ID17) address two different aspects of the problem, and by associating property `dct:accessRights` with `dcat:Dataset` and `dcat:Distribution`, respectively. However, both cases highlight that DCAT misses a property to specify access restrictions / conditions - since `dct:license` is about *use* conditions.

About how to express access conditions / restrictions:
- for simple / standardised access restrictions - analogous to the existing solutions mentioned in the use case - URIs can be used
- for access restrictions not fitting in the former ones, ODRL could be an option - so this links to the discussion around #114

**Comment by andrea-perego** [comment]
About ODRL and access restrictions, the POE WG has recently published on GH an [ODRL Best Practices](http://w3c.github.io/poe/bp/) document, where one of the examples concerns [_Representation of access policies to digital resources_](http://w3c.github.io/poe/bp/#example-2-representation-of-access-policies-to-digital-resources).

**Comment by jakubklimek** [comment]
I tried to play with ODRL & DCAT, but I am not entirely sure about the alignment. Could we create an example of, e.g., describing a dataset where one distribution is published under CC0 and one under CC BY?

My initial draft with one distribution - comments welcome:
```turtle
@prefix dcterms: <http://purl.org/dc/terms/> .
@prefix dcat:    <http://www.w3.org/ns/dcat#> .
@prefix odrl:    <http://www.w3.org/ns/odrl/2/> .
@prefix cc:      <http://creativecommons.org/ns#> .

<dataset> a dcat:Dataset, odrl:AssetCollection;
  dcat:distribution <distribution> ;
  dcterms:publisher <publisher> .
  
<distribution> a dcat:Distribution, odrl:Asset ;
  dcterms:rights <distribution/rights> ;
  odrl:partOf <dataset> ;
  odrl:hasPolicy <distribution/rights> .

<distribution/rights> a dcterms:RightsStatement, odrl:Policy, odrl:Set ;
  odrl:permission <distribution/rights/permissions> .
  
<distribution/rights/permissions> a odrl:Permission ;
  odrl:target <distribution> ;
  odrl:assigner <publisher> ;
  odrl:action cc:Sharing, cc:Distribution, odrl:reproduce, odrl:distribute, odrl:modify  .
  ```

**Comment by makxdekkers** [comment]
@jakubklimek My proposal was slightly different. I suggested that a publisher only use `odrl:hasPolicy` iff the object is an `odrl:Policy`. In case the object is a CC licence, they would use `dct:license`.

```
<distribution> a dcat:Distribution ;
  dcterms:license <https://creativecommons.org/licenses/by/4.0/>  .
```

**Comment by jakubklimek** [comment]
@makxdekkers My question was related more to the connection between DCAT and ODRL in general than the specifics of linking CC licences, i.e. whether `dcat:Dataset` could be `odrl:AssetCollection`, `dcat:Distribution` an `odrl:Asset`, etc.

Specifically for CC licences, your proposal may be more straightforward. At the same time there is an [ODRL Creative Commons Profile](https://www.w3.org/community/odrl/work/cc/#section-References) which I used in the example, which then allows users to search e.g. for all datasets with "distribution permission", which is not possible if the CC license is only linked and not described by ODRL.

**Comment by makxdekkers** [comment]
@jakubklimek Yes, the additional functionality you mention, searching for specific permissions, would be useful. However, from my experience, it is already a challenge to get people to do the very simple things, like linking to a CC licence, so I am just wondering how much detailed information about permissions, obligations etc. we can hope to see in practice.
Having said that, whenever someone has linked to a CC licence, a lot of information about permissions and obligations is implicit in the CC licence, so a smart catalogue system could maybe generate (some aspects of) an ODRL policy out of a CC licence.
But other than that, I have no problem with your example. Thanks.

**Comment by dr-shorthair** [comment]
Currently we have dct:accessRights associated with Services. 
Also needs to be on Distributions? 
See #59

**Comment by aisaac** [comment]
If dct:accessrights can be used on Services, I don't see why it wouldn't be possible to use it with Distributions (or even with Datasets, but this is another battle that has been fought in other issues I believe ;-) )

**Comment by davebrowning** [comment]
There was a battle re; access rights at the dataset level?

**Comment by aisaac** [comment]
There was a battle on rights in general: https://github.com/w3c/dxwg/issues/104

**Comment by agbeltran** [comment]
Addressed in #607
